rust/scx_utils/src/topology.rs

```diff
@@ -150,13 +158,26 @@ impl Cpu {
     pub fn llc_id(&self) -> usize {
         self.llc_id
     }
+
+    /// Returns true if the max frequency of the cpu is higher than the average frequency of all
+    /// cores.
+    pub fn is_pcore(&self) -> bool {
+        self.core_type == CoreType::PCore
+    }
```

Comment:
Can we just `pub` these fields rather than keeping adding trivial accessors? Not sure what these add.